https://code.djangoproject.com/ticket/34118#comment:11
https://github.com/python/cpython/issues/102549
https://github.com/django/django/commit/f55bcff9dcac1d0cb1f927f9fd543cfd567400c4

From f55bcff9dcac1d0cb1f927f9fd543cfd567400c4 Mon Sep 17 00:00:00 2001
From: Mariusz Felisiak <felisiak.mariusz@gmail.com>
Date: Fri, 7 Apr 2023 11:07:54 +0200
Subject: [PATCH] [4.1.x] Refs #34118 -- Fixed
 CustomChoicesTests.test_uuid_unsupported on Python 3.11.4+.

https://github.com/python/cpython/commit/5342f5e713e0cc45b6f226d2d053a8cde1b4d68e

Follow up to 38e63c9e61152682f3ff982c85a73793ab6d3267.

Backport of 2eb1f37260f0e0b71ef3a77eb5522d2bb68d6489 from main
--- a/tests/model_enums/tests.py
+++ b/tests/model_enums/tests.py
@@ -311,8 +311,7 @@ class Timezone(datetime.timezone, models.Choices):
                 pass
 
     def test_uuid_unsupported(self):
-        msg = "UUID objects are immutable"
-        with self.assertRaisesMessage(TypeError, msg):
+        with self.assertRaises(TypeError):
 
             class Identifier(uuid.UUID, models.Choices):
                 A = "972ce4eb-a95f-4a56-9339-68c208a76f18"
 
